README

```diff
@@ -1548,16 +1549,18 @@ To display the DOSBox status window:
 =====================================
 
 The configuration file is automatically created the first time you run DOSBox.
-The file can be found in:
-   (Windows)  "Start/WinLogo Menu"->"All Programs"->"DOSBox-0.74-2"->Options
-   (Linux)    ~/.dosbox/dosbox-0.74-2.conf
-   (Mac OS X) "~/Library/Preferences/DOSBox 0.74-2 Preferences"
+
+The default directories storing config file are:
+
+   (Windows) C:\Users\<username>\AppData\Local\DOSBox\
+   (Linux)   ~/.config/dosbox/
+   (macOS)   ~/Library/Preferences/DOSBox/
+
 The file is divided into several sections. Each section starts with a
 [section name] line. The settings are the property=value lines where value can
 be altered to customize DOSBox.
 # and % indicate comment-lines.
```

Comment:
Suggest mentioning XDG here, and how the user can use or be affected by it.  I've made a rough attempt below:
```
   (Linux)   ~/.config/dosbox/
   (macOS)   ~/Library/Preferences/DOSBox/

Linux users: the configuration's parent directory can be customized by setting the XDG_DATA_HOME environment variable, per the XDG Base Directory Specification. If set, DOSBox will look $XDG_DATA_HOME/dosbox/ for its configuration file.  

The file is divided into several sections. Each section starts with a
```